src/controller/controller.c

```diff
@@ -1051,6 +1245,7 @@ static int controller_property_get_log_target(
 static const sd_bus_vtable controller_vtable[] = {
         SD_BUS_VTABLE_START(0),
         SD_BUS_METHOD("ListUnits", "", NODE_AND_UNIT_INFO_STRUCT_ARRAY_TYPESTRING, controller_method_list_units, 0),
+        SD_BUS_METHOD("ListUnitFiles", "", "a(sss)", controller_method_list_unit_files, 0),
```

Comment:
Please extend the D-Bus specification for the 
- Controller: https://github.com/eclipse-bluechi/bluechi/blob/main/data/org.eclipse.bluechi.Controller.xml
- and the Node: https://github.com/eclipse-bluechi/bluechi/blob/main/data/org.eclipse.bluechi.Node.xml
by the new `ListUnitFiles` method. You can use the `ListUnits` in these files as a reference. 

After updating the xml files, please run:
```bash
# install dependencies
$ python3 -m ensurepip --default-pip
$ python3 -m pip install -r requirements.txt
$ python3 -m pip install -r src/bindings/generator/requirements.txt

# generate python bindings with new api method list unit files
$ build-scripts/generate-bindings.sh python
```